File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT2_JAVA/src/Organisms/Plants/Dandelion.java
package Organisms.Plants;

import Game.World;

import java.awt.*;
import java.io.Serializable;

public class Dandelion extends Plant implements Serializable {
    public Dandelion(World world, int x, int y)
    {
        super("D", "Dandelion", Color.yellow, 0, 0, 1, world,x,y);
    }
    @Override
    public Dandelion getChild(int x, int y)
    {
        return new Dandelion(world,x,y);
    }
    @Override
    public void Action()
    {
        for(int i=0;i<3;i++)
        {
            super.Action();
        }
    }
}

File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT3_PYTHON/Organisms/Plants/dandelion.py
from Organisms.Plants.plant import Plant


class Dandelion(Plant):
    def __init__(self, position_x, position_y, world):
        super().__init__(position_x, position_y, "Dandelion", 'D', 0, 0, world, 0, (255,255,0))

    def getchild(self, newposition):
        return Dandelion(newposition[0], newposition[1], self.world)

    def action(self):
        for i in range(0,3):
            super().action()
